package/wpe/wpeframework-plugins/Config.in

```diff
@@ -64,6 +64,23 @@ config BR2_PACKAGE_WPEFRAMEWORK_DEVICEINFO
 	help
 	  DeviceInfo Plugin
 
+config BR2_PACKAGE_WPEFRAMEWORK_DSRESOLUTION
+    select BR2_PACKAGE_WPEFRAMEWORK_PLUGINS
+    bool "DSResolution"
+    default y
```

Comment:
Lets not by default add this plugin. It needs to be sleected y the customer as this is not something most customers are looking for

Reply:
Updated.